sklearn/linear_model/tests/test_omp.py

```diff
@@ -27,6 +27,7 @@
     n_features=n_samples,
     n_nonzero_coefs=n_nonzero_coefs,
     random_state=0,
+    data_transposed=True,
```

Comment:
Do we want to change the test to deal now with the change of shape?

Reply:
I think we can leave that to when we deprecate the param